cmake/modules/FindPython.cmake

```diff
@@ -29,11 +29,12 @@ else()
   list(APPEND PYTHON_LIBRARIES ${PC_PYTHON_STATIC_LIBRARIES})
 endif()
 
+list(APPEND PYTHON_DEFINITIONS -DHAS_PYTHON=1)
 
 include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(PYTHON REQUIRED_VARS PYTHON_INCLUDE_DIR PYTHON_LIBRARY PYTHON_LIBRARIES)
+find_package_handle_standard_args(PYTHON REQUIRED_VARS PYTHON_INCLUDE_DIR PYTHON_LIBRARY PYTHON_LIBRARIES PYTHON_DEFINITIONS)
```

Comment:
since we already require python(libs) by the time we hit this, i figured the additional sanity check wouldn't hurt in case some enterprising individual removed it by accident/stupidity. removed.